abs.yaml

```diff
@@ -0,0 +1,2 @@
+channels:
```

Comment:
Is this a dev/debug file?

Reply:
Yes, this is a dev file with a staging channel intended to take the dependency up. I'll remove it before merging.